src/DynamoCore/Engine/CodeCompletion/CodeCompletionServices.cs

```diff
@@ -69,7 +69,7 @@ internal IEnumerable<CompletionData> GetCompletionsOnType(string code, string st
                 }
             }
 
-            return members?.Select(x => CompletionData.ConvertMirrorToCompletionData(x));
```

Comment:
So if I understand correctly, `members` currently contains all methods from the given class as well as those from the base class, correct?

Reply:
Then what happens when you autocomplete the method by typing `(`? Does it show method signatures for only the hiding methods and not the hidden methods?

Reply:
> `members` currently contains all methods from the given class as well as those from the base class, correct?

Correct

Reply:
> Then what happens when you autocomplete the method by typing `(`? Does it show method signatures for only the hiding methods and not the hidden methods?

It will show all signatures except the hidden ones